src/Application.php

```diff
@@ -48,4 +55,14 @@ public function doRun(InputInterface $input, OutputInterface $output): int
 
         return parent::doRun($input, $output);
     }
+
+    public function getConfig(): ?Config
+    {
+        return $this->config;
+    }
+
+    public function setConfig(string $configDir): void
```

Comment:
Isn't it weird, that `getConfig` returns a `Config` object while `setConfig` takes a `string`? I would suggest to name the method `resolveAndLoadConfig` which in turn calls `setConfig(Config $config)`.

If so, the same holds for `\Slim\Console\Command\AbstractCommand`.

Reply:
I agree with @adriansuter